service_container/injection_types.rst

```diff
@@ -100,6 +100,112 @@ working with optional dependencies. It is also more difficult to use in
 combination with class hierarchies: if a class uses constructor injection
 then extending it and overriding the constructor becomes problematic.
 
+Immutable-setter Injection
+--------------------------
+
```

Comment:
And I would propose a versionadded directive here

Reply:
Same as before.

Reply:
Please add a blank line after the directive and rephrase it like:

... was introduced in Symfony 4.3.

Thank you

Reply:
Fixed 🙂